app/templates/index.hbs

```diff
@@ -16,8 +16,10 @@
                 <div class="preprint-search col-xs-10 col-xs-offset-1 col-md-8 col-md-offset-2 m-v-lg" >
                     {{search-preprints search="search"}}
                     <p class="p-l-md">
-                        <span class="f-w-lg">{{model.preprints.length}}</span> preprints as of
-                        <span>{{moment-format model.theDate "MMMM DD, YYYY"}}</span>
+                        <span class="f-w-lg">{{model.preprints.length}} hosted preprints, </span>
```

Comment:
If you have more than 10 results, how many does `model.preprints.length` display here? Ember data only fetches the first page of results for relationships used this way, so you may get an incorrect number. `links.meta.total` can be a good, API-provided alternative way of getting the more accurate count.